Algorithm.Python/readme.md

```diff
@@ -55,31 +77,23 @@ conda install -y cython pandas
  2. Run LEAN. You should see the same result of the C# algorithm you tested earlier.
 ___
 #### Python.Runtime.dll compilation
-LEAN users do **not** need to compile Python.Runtime.dll. The information below is targeted to developers who wish to improve it. 
+LEAN users do **not** need to compile `Python.Runtime.dll`. The information below is targeted to developers who wish to improve it. 
 
 Download [QuantConnect/pythonnet](https://github.com/QuantConnect/pythonnet/) github clone or downloading the zip. If downloading the zip - unzip to a local pathway.
 
-**Note:** QuantConnect's version of pythonnet is an enhanced of [pythonnet](https://github.com/pythonnet/pythonnet) with support to System.Decimal and System.DateTime.
+**Note:** QuantConnect's version of pythonnet is an enhanced of [pythonnet](https://github.com/pythonnet/pythonnet) with support to `System.Decimal` and `System.DateTime`.
```

Comment:
is an enhanced *version* of pythonnet ... with *added* support *for* `System.Decimal` and `System.DateTime`